src/Microsoft.IdentityModel.Tokens/Exceptions/SecurityTokenException.cs

```diff
@@ -74,5 +75,22 @@ protected SecurityTokenException(SerializationInfo info, StreamingContext contex
             : base(info, context)
         {
         }
+
+#if NETSTANDARD2_0
```

Comment:
I don't follow, GetObjectData is available in the lowest target

https://docs.microsoft.com/en-us/dotnet/api/system.exception.getobjectdata?view=netframework-4.5

Reply:
Right but for .net CORE there are no /// comments, so the inheritdoc has nothing to inherit from.
You can convince yourself by removing the method and building the branch. SauceControl will show a warning just for our netcore build.